src/nemo_evaluator/executors/slurm_executor.py

```
@@ -527,6 +559,154 @@ def run(self, config, *, dry_run=False, resume=False, background=False, **_kwarg
+        from nemo_evaluator.executors.ssh import _ssh, apply_srtctl
+
+        # srtctl mounts {srtctl_work_dir}/configs/ as /configs inside the
+        # benchmark container — place inner config and credentials there.
+        remote_configs = f"{srtctl_work_dir}/configs"
+        recipe_filename = f"nel_recipe_{os.urandom(4).hex()}.yaml"
+        remote_recipe = f"{srtctl_work_dir}/{recipe_filename}"
+
+        _ssh(target, f"mkdir -p {remote_configs}")
+        # Pyxis requires the bind-mount source (output.dir's parent) to exist.
+        _ssh(target, f"mkdir -p {shlex.quote(config.output.dir)}")
+        subprocess.run(
```

Ruff (0.15.14)

[error] `subprocess` call: check for execution of untrusted input

(S603)